### DISCUSSION TITLE
Default robots.txt for all websites

### DISCUSSION THREAD

**Discussion by chosso** [NginxProxyManager/nginx-proxy-manager#3468] [Q&A]
Hi,

how can I implement the robots.txt for all websites managed my NPM?

I use this snippet:
```
location = /robots.txt {
  add_header  Content-Type  text/plain;
  return 200 "User-agent: *\nDisallow: /\n";
}
```

inside custom .conf like mentioned here: https://nginxproxymanager.com/advanced-config/#custom-nginx-configurations 
but it errors out `/data/nginx/custom/root.conf`: 
```
nginx: [emerg] "location" directive is not allowed here in /data/nginx/custom/root.conf:1
```

As of now, I implemented the snippet to the _Custom Nginx Configuration_ of the _Advanced Configuration_ for each! side. This is kinda tedious in the long run.

I'm using NPM inside Docker if this info is needed.

**Comment by ioqy** [comment]
Add the location directive to `server_proxy.conf` in the [custom config directory](https://nginxproxymanager.com/advanced-config/#custom-nginx-configurations) and then restart NPM.
